src/Model/AuditReaderInterface.php

```diff
@@ -19,46 +19,28 @@
 interface AuditReaderInterface
 {
     /**
-     * @param string $className
-     * @param string $id
-     * @param string $revision
-     *
-     * @return object
+     * @param mixed $id
```

Comment:
We cant have a better type than mixed ?

Reply:
I dont really know, I am following the implementation on doctrineOrm that just uses this: https://github.com/simplethings/EntityAuditBundle

Reply:
I dont think about something else than Int|string for an id.
But if we cant be sure thats not a blocking point

Reply:
Uuid is another example (object)

Reply:
I would prefer to leave it as mixed and do a second round finding the mixed and deciding if they should be changed or not.

Reply:
ok